flask_admin/contrib/mongoengine/__init__.py

```diff
@@ -0,0 +1,8 @@
+# flake8: noqa
+try:
+    import mongoengine
+except ImportError:
+    raise Exception("Please install mongoengine in order to use mongoengine backend")
```

Comment:
Please can we copy the error format from other extensions, which will include adding a reference to `flask-admin[mongoengine]` - and setting that up as an extra in `pyproject.toml`.